modules/core/src/main/java/org/apache/synapse/FaultHandler.java

```diff
@@ -135,4 +143,23 @@ private void traceOrDebugWarn(boolean traceOn, String msg) {
         }
         log.warn(msg);
     }
+
+    /**
+     * Execute the ExtendedSynapseHandler in the error flow.
+     *
+     * @param synCtx Synapse Message Context
+     */
+    protected void executeExtendedSynapseHandler(MessageContext synCtx) {
```

Comment:
Since we are executing only error handling , let's rename this to reflect it ?

Reply:
Fixed